src/backend/feed/processor.js

```diff
@@ -141,7 +141,8 @@ module.exports = async function processor(job) {
           logger.warn(`${info.status} Feed no longer available: ${feed.url}`);
           break;
         case 429:
-          logger.warn(`${info.status} Feed requested too many times: ${feed.url}`);
+          logger.warn(`${info.status} Feed requested too many times, setting delay: ${feed.url}`);
+          feed.setDelayed(15);
```

Comment:
Let's make this configurable in the env, and default to 60 seconds * 60 minutes = 1 hour (3600s)

Reply:
Sure, added a `FEED_PROCESSING_DELAY_SEC` variable to `env.example`: https://github.com/Seneca-CDOT/telescope/pull/726/commits/850b45fd82e7ffcf6bcc1c91ecc3b4b9e7efd949